include/exception.h

```diff
@@ -105,6 +105,21 @@ typedef struct
     volatile reg_block_t* regs;
 } exception_t;
 
+
+/** 
+ * @brief Guaranteed length of the reset time.
+ * 
+ * This is the guaranteed length of the reset time, that is the time
+ * that goes between the user pressing the reset button, and the CPU actually
+ * resetting. See #exception_reset_time for more details.
+ * 
+ * @note The general knowledge about this is that the reset time should be
+ *       500 ms. Testing on different consoles show that, while most seem to
+ *       reset after 500 ms, a few EU models reset after 200ms. So we define
```

Comment:
Is the timer on the console or the cartridge? I remember a discussion about it being implemented on the CIC chip. If your dev cartridges are using 3rd party chips that implement the timer differently, it might be only like this for your carts.

Reply:
I think this is pending further investigations, there are many question marks. I myself don't fully believe this but it is what has been measured. 

This PR has been pending these investigations for a long time, and I don't know when they will be done. Meanwhile, the reset handling on trunk is broken (we show an exception screen).

My suggestion is to go through with this as-is and then we can change this specific macro once we fully understand the situation. Meanwhile we can grow code that does the right thing, using this macro for calculations so that it will adapt to new discoveries.

If you prefer I can change this to 500 and ignore the findings until we better frame them.

Reply:
Anything under 500ms is safe so we can continue as is 👍🏼 I can review the code in general soon but on a quick glance it all looks good.